src/main/java/dk/cngroup/lentils/service/GameLogicService.java

```diff
@@ -49,16 +49,16 @@ public boolean passedAllCyphers(final Team team) {
 
     public boolean passedTimeToViewFinalPlace() {
         try {
-            LocalDateTime finalPlaceOpeningTime = finalPlaceService.getFinalPlace().getOpeningTime();
-            return finalPlaceOpeningTime.isBefore(LocalDateTime.now().plusHours(1));
+            LocalDateTime finalPlaceResultsTime = finalPlaceService.getFinalPlace().getResultsTime();
```

Comment:
Tady bych to asi kontroloval oproti finishTime.

Kdyby náhodou někdo nastavil rozestup mezi finish a results time na hodinu. Tak aby i tak se to hráčům zobrazilo.

Reply:
makes sense :) díky